docs/src/features.md

```diff
@@ -272,7 +302,7 @@ This section quantifies the advantages of the [`flexible connection`](@ref flex-
 The table below shows the number of constraints and variables for each approach over a six-hour horizon. This highlights the potential of flexible time resolution in reducing the size of the optimization model.
 
 | Modeling approach                          | Nº Variables | Nº Constraints | Objective Function |
-|:-------------------------------------------|:------------ |:---------------|:------------------ |
+| :----------------------------------------- | :----------- | :------------- | :----------------- |
```

Comment:
This section also needs to be updated since the new formulation has a different number of variables and constraints than what we used to have. For instance, if you run the `Variable Resolution` case study with the flexible connection and time resolution, you get the following:

```julia
Optimize a model with 29 rows, 16 columns
```

The objective function remains the same

Reply:
Thank you for the heads-up.